app/models/fake_solr_document.rb

```diff
@@ -37,7 +37,7 @@ def initialize(options = {})
         Faker::Number.unique.within(range: 1..1000)
       ],
       "file_name_ssim": [
-        'thesis_1.pdf'
+        options[:file_name] || 'thesis_1.pdf'
```

Comment:
i wonder if we should make `file_name` into `file_names` and smash it in kind of like we do with keywords? we may run into a case where we want to test multiple files?

Reply:
I'll add this to this PR

Reply:
![Screen Shot 2022-06-23 at 10 51 40 AM](https://user-images.githubusercontent.com/32677188/175328794-e6cfd5dc-99ad-4f1f-8682-68c4780e13b5.png)
![Screen Shot 2022-06-23 at 10 51 46 AM](https://user-images.githubusercontent.com/32677188/175328809-83a589d7-45c9-4754-a1a1-296fb22639ad.png)
Displays nicely 👍